src/Lorentz/Contracts/BaseDAO.hs

```diff
@@ -65,3 +70,17 @@ ensureZeroTransfer = do
   amount
   push zeroMutez
   if IsEq then nop else failCustom_ #fORBIDDEN_XTZ
+
+pushFuncContext
+  :: (KnownValue pm)
+  => Proxy pm
```

Comment:
Do we really need to pass `Proxy` if we have type applications?

Reply:
I don't like when mandatory arguments are passed via just `TypeApplications` because, when the argument is missing, this causes slightly unclear errors like `Cannot deduce 'instance IsoValue pm0'`. But ok, this method is used only in one place anyway, so there should be no problem in removing `Proxy`.